### DISCUSSION TITLE
Relax username requirements or consider using email addresses instead

### DISCUSSION THREAD

**Discussion by ctgraham** [pkp/pkp-lib#7965] [Proposals]
What is driving the requirement that "The username must contain only lowercase letters, numbers, and hyphens/underscores."?

With Shibboleth authentication, the best practice username would be a scoped attribute, eg. user@institution-scope (usually this is just an email address).

Are there any known dependencies that will break if we loosen the username validation?

**Comment by asmecher** [comment]
No, nothing should depend on the username having specific content. FWIW, we might also reconsider our separation of usernames and emails; I've fought against this before but I'll concede that I find that approach useful.

**Comment by asmecher** [comment]
(See essentially the same proposals at https://github.com/pkp/pkp-lib/issues/4053)

**Comment by kabatovam** [comment]
Hello, I wanted to ask if there was any work done with regards to this issue. From what I checked this is not solved, but the original issue 906 was defined as closed. 

We started using OJS and our first user had problems registering because of this error. He thought there was an issue with his password...

I would like to know is if there is a way to change up the error message because it seems to be confusing to users.

**Reply by NateWr** [reply]
No work has been done so far. #906 was only closed because we converted into this discussion.

If you have a proposal to make about the error message, please share info about what the error message is now and what you think it should be.

**Comment by amandasramalho** [comment]
We started testing the OMP with a Brazilian publisher indexed in SciELO and that was the first question. This implementation would be very useful.

**Comment by asmecher** [comment]
This has been resolved via https://github.com/pkp/pkp-lib/issues/8871.
